### PR TITLE
feat: make CNI plugin to work with Kuma

### PR DESCRIPTION
**Pull request by bartsmykla**
Adjusted CNI configuration to provide argument with the type
of service mesh merbridge is working with and updated code
to make CNI plugin work with Kuma.

+ Adjusted tests for plugin
+ added mac's `.DS_store` files to `.gitignore` file

### CODE REVIEW

File: pkg/cniplugin/plugin_config.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package cniplugin
```
Comment (kebe7jun) [inline pkg/cniplugin/plugin_config.go:1]: Need add license header.

**Review by kebe7jun — COMMENTED** [review]
almost lgtm.

File: pkg/cniplugin/plugin_config.go (hunk 1 above)
Comment (bartsmykla) [reply pkg/cniplugin/plugin_config.go:1]: On it

Comment (bartsmykla) [reply pkg/cniplugin/plugin_config.go:17]: done

**Comment by bartsmykla** [comment]
Working on style-check

File: app/cmd/root.go [hunk 2]
```diff
@@ -107,8 +107,8 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&config.Context, "kubecontext", "", "The name of the kube config context to use")
 }
 
-func installCNI(ctx context.Context, cniReady chan struct{}) {
-	installer := cniserver.NewInstaller()
+func installCNI(ctx context.Context, serviceMeshMode string, cniReady chan struct{}) {
```
Comment (dddddai) [inline app/cmd/root.go:110]: Why not just use the global `config.Mode`?

**Review by dddddai — COMMENTED** [review]
Thanks!

File: app/cmd/root.go (hunk 2 above)
Comment (bartsmykla) [reply app/cmd/root.go]: changed according to your suggestion! :-)

Comment (dddddai) [reply app/cmd/root.go]: Sorry I wasn't clear, I mean directly reading `config.Mode` where needed rather than passing the value

**Review by kebe7jun — COMMENTED** [review]
lgtm if @dddddai approve.

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**